HardcodeTray/modules/svg/inkscape.py

```diff
@@ -35,10 +35,33 @@ def __init__(self, colors):
 
     def convert_to_png(self, input_file, output_file, width=None, height=None):
         """Convert svg to png."""
-        cmd = [self.cmd, "-z", "-f", input_file, "-e", output_file]
+
+        is_out_of_beta = any(
+            [
+                b"." in x and int(x[: x.find(b".")]) >= 1
+                for x in execute([self.cmd, "--version"], verbose=False).split()
+            ]
+        )
+
+        if is_out_of_beta:
+            cmd = [
+                self.cmd,
+                "--export-area-drawing",
+                "--export-type=png",
+                input_file,
+                "-o",
+                output_file,
+            ]
+        else:
+            cmd = [self.cmd, "-z", "-f", input_file, "-e", output_file]
 
         if width and height:
-            cmd.extend(["-w", str(width), "-h", str(height)])
+            insert_pos = 1
+            to_insert = ["-w", str(width), "-h", str(height)]
+
+            for arg in to_insert:
+                cmd.insert(insert_pos, arg)
+                insert_pos += 1
```

Comment:
Any reason for having to do it this way instead of what used to be done? I don't get the need of having to add the arguments at the beginning of the cmd

Reply:
Well I did it wrong but my objective was to follow their manual.

```
"inkscape [options] [filename_1 filename_2 ...]
```